docker/docker-compose.yaml

```diff
@@ -8,32 +8,32 @@ services:
         target: /app
     build: 
       dockerfile: ../docker/Dockerfile.yew
-    command: bash -c "cd app/yew-ui && trunk serve --address 0.0.0.0"
+    command: bash -c "cd app/yew-ui && trunk serve --address 0.0.0.0 --port ${TRUNK_SERVE_PORT:-80}"
     environment:
-      - ACTIX_PORT=8080 
-      - TRUNK_SERVE_PORT=80
+      - ACTIX_PORT=${ACTIX_PORT:-8080}
+      - TRUNK_SERVE_PORT=${TRUNK_SERVE_PORT:-80}
```

Comment:
@resnickmicah this is awesome!! 

I did not know you could provide defaults like this!!!